@here/harp-mapview/lib/MapView.ts

```diff
@@ -3384,6 +3384,7 @@ export class MapView extends THREE.EventDispatcher {
                 Math.floor(this.zoomLevel),
                 this.getEnabledTileDataSources(),
                 this.m_frameNumber,
+                this.taskQueue,
```

Comment:
As the visibleTileSet is basically not working without a taskQueue and it will always be the same taskQueue passed through here, I would prefer adding it as a hard dependency in the constructor of VisibleTileSet, instead of passing it here every frame.

Reply:
Good point! Have fixed it.